cdk-project/lib/images/processing-image/execute.py

```diff
@@ -65,6 +65,7 @@ def run_notebook():
         if not kernel:
             nb_kernel = kernel_for(notebook_file)
             avail_kernels = available_kernels()
+            print("available kernels are ", avail_kernels)
```

Comment:
Did this slip in while debugging or did you intend to add it? If you are adding this intentionally let's make it more informative and also add the requested nb_kernel.

Reply:
This was intentional, I will make it more descriptive, thanks!